cdp_scrapers/legistar_utils.py

```diff
@@ -109,3 +180,858 @@ def get_legistar_events_for_timespan(
 
     log.debug(f"Collected {len(response)} Legistar events")
     return response
+
+
+def stripped(in_str: str) -> str:
+    """
+    Return leading and trailing whitespace removed if it is a string
+
+    Parameters
+    ----------
+    in_str : str
+
+    Returns
+    -------
+    str or Any
+        in_str stripped if it is a string
+    """
+    if isinstance(in_str, str):
+        return in_str.strip()
+    return in_str
+
+
+def reduced_list(in_list: List) -> List:
+    """
+    Remove all None items from in_list
+
+    Parameters
+    ----------
+    in_list : List
+
+    Returns
+    -------
+    List
+    """
+    return [item for item in in_list if item is not None]
```

Comment:
I had to change this to:

```python
def reduced_list(in_list: List) -> List:
    """
    Remove all None items from in_list

    Parameters
    ----------
    in_list : List

    Returns
    -------
    List
    """
    filtered = [item for item in in_list if item is not None]
    if len(filtered) == 0:
        return None
    
    return filtered
```

Reply:
Gotcha. I did have the `len(filtered)` check in a previous version.

That's just me misreading your comment from before, where you suggested `[item for item in in_list if item is not None]`. I thought you meant to just do that, only, haha. Will change.

Reply:
Yea I think earlier me forgot about later filters